Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -42,98 +136,110 @@ struct DashboardView: View {
                 }
             }
 
-            VStack(spacing: 15) {
-                if let loggedInUserEmail = userManager.email {
-                    Text(loggedInUserEmail)
-                }
-                Text("What would you like to test?")
-
-                Group {
-                    ColorButton("Send Random Event") {
-                        switch Int.random(in: 0 ..< 3) {
-                        case 0:
-                            CustomerIO.shared.track(name: "Order Purchased")
-                        case 1:
-                            CustomerIO.shared.track(
-                                name: "movie_watched",
-                                properties: [
-                                    "movie_name": "The Incredibles"
-                                ]
-                            )
-                        default: // case 2
-                            CustomerIO.shared.track(
-                                name: "appointmentScheduled",
-                                properties: [
-                                    "appointmentTime": Calendar.current.date(byAdding: .day, value: 7, to: Date())!.epochNoMilliseconds
-                                ]
-                            )
-                        }
-
-                        nonBlockingMessage = "Random event sent"
+            ScrollView {
+                VStack(spacing: 15) {
+                    if let loggedInUserEmail = userManager.email {
+                        Text(loggedInUserEmail)
                     }
-                    .setAppiumId("Random Event Button")
-
-                    ColorButton("Send Custom Event") {
-                        subscreenShown = .customEvent
-                    }.setAppiumId("Custom Event Button")
-                        .sheet(isPresented: .constant(subscreenShown == .customEvent), onDismiss: { subscreenShown = nil }) {
-                            CustomEventView(close: {
-                                subscreenShown = nil
+                    Text("What would you like to test?")
+                    Group {
+                        // ---- In-app Inline View ----
+                        InAppMessageViewRepresentable(elementId: "dashboard-announcement", containerWidth: $containerWidth, heightTracker: heightTracker)
+                            // Set the height of the view based on the height tracked
+                            .frame(height: heightTracker.subviewHeight)
+                            // GeometryReader to track the size of the container
+                            .background(GeometryReader { geometry in
```

Comment:
`background` adds a background color/view behind a view. In this case we are adding a clear color and when it appears it calculates the width of the view and updates it.